sync/utils.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 The Casdoor Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sync
+
+import (
+	"bytes"
+	"log"
+
+	"github.com/xorm-io/xorm"
+)
+
+func GetUpdateSql(schemaName string, tableName string, columnNames []string, newColumnVal []string, oldColumnVal []string) string {
+	var bt bytes.Buffer
+	bt.WriteString("update " + schemaName + "." + tableName + " set ")
```

Comment:
fmt.Printf, fmt.Sprintf

Reply:
Or should we use a SQL builder?

https://www.reddit.com/r/golang/comments/a7usrp/any_decent_sql_query_builders_out_there/
https://github.com/Masterminds/squirrel